voorhoede/tudelft-spacefinder
# Feat/implement new navigation design

# Changes

Implements the new mobile navigation as designed:
- Add a navigation bar to the bottom of the screen.
- Remove items that are now visible in the navigation bar from the header menu.
- Move filter button from header to navigation bar.
- Show number of selected filters in the filter button.
- Add pages for the map view, list view and building view on mobile; keep list and building view on desktop.
- Refactor button classes to make them also usable for links.
- Moves the map loading state outside of mapbox div, as this div must remain empty.
- Remove zoom level buttons from map.

# Associated issue

Resolves:
- https://trello.com/c/A0pwVIaM/89-create-navigation-on-mobile-and-desktop-according-to-design
- https://trello.com/c/yAetIqHJ/53-improving-visibility-of-filters
- https://trello.com/c/oDSQknRf/55-active-indicator-of-filters-more-prominent
- https://trello.com/c/Kv65E2Us/92-change-copy-of-filter-overview-to-mention-filters-below-instead-of-above

# How to test

1. Open the preview link on both desktop and mobile.
2. Check that the navigation is added to the bottom of the screen (mobile) or column (desktop).
3. See if the pages show the correct information.
4. See if the filters still works; number of selected filters should be visible in the filter button.

# Checklist
